nix/all-engines.nix

```diff
@@ -128,19 +128,26 @@ in
     ({ profile }: stdenv.mkDerivation {
       name = "query-engine-wasm-gz";
       inherit src;
+      buildInputs = with pkgs; [ iconv ];
 
       buildPhase = ''
       export HOME=$(mktemp -dt wasm-engine-home-XXXX)
       
       OUT_FOLDER=$(mktemp -dt wasm-engine-out-XXXX)
       ${self'.packages.build-engine-wasm}/bin/build-engine-wasm "0.0.0" "$OUT_FOLDER" 
-      gzip -ckn "$OUT_FOLDER/query_engine_bg.wasm" > query_engine_bg.wasm.gz
+
+      for provider in "postgresql" "mysql" "sqlite"; do
+        gzip -ckn "$OUT_FOLDER/$provider/query_engine_bg.wasm" > "$provider.wasm.gz"
```

Comment:
should we prefix this with `query-engine-`?

Reply:
Done